packages/connectors/connector-oidc/src/constant.ts

```diff
@@ -46,6 +47,56 @@ export const defaultMetadata: ConnectorMetadata = {
       required: true,
       placeholder: '<client-secret>',
     },
+    {
+      key: 'tokenEndpointAuthMethod',
+      label: 'Token Endpoint Auth Method',
+      type: ConnectorConfigFormItemType.Select,
+      selectItems: [
+        {
+          title: TokenEndpointAuthMethod.ClientSecretPost,
+          value: TokenEndpointAuthMethod.ClientSecretPost,
+        },
+        {
+          title: TokenEndpointAuthMethod.ClientSecretBasic,
+          value: TokenEndpointAuthMethod.ClientSecretBasic,
+        },
+        {
+          title: TokenEndpointAuthMethod.ClientSecretJwt,
+          value: TokenEndpointAuthMethod.ClientSecretJwt,
```

Comment:
can we reuse oauth2 connector as much as possible? e.g. reuse the config object. non-blocking though, you can add an issue to do it later

Reply:
Updated.